From 7a6f3989a16993112a3e3477e67cb81c602ae571 Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Thu, 19 Apr 2012 15:54:40 +0800
Subject: [PATCH 316/435] ENGR00180239-2: Add command help for
 dump-clocks-dot.sh script

As title, it's just a help, easy to refresh our mind.

Signed-off-by: Robby Cai <R63905@freescale.com>
---
 test/mxc_clock_test/dump-clocks-dot.sh | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/test/mxc_clock_test/dump-clocks-dot.sh b/test/mxc_clock_test/dump-clocks-dot.sh
index b325947..adba4b9 100755
--- a/test/mxc_clock_test/dump-clocks-dot.sh
+++ b/test/mxc_clock_test/dump-clocks-dot.sh
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+if [ $# -gt 0 ]; then
+	echo "Usage:"
+	echo "	1. run '$0 > d.txt' on Target Board"
+	echo "	2. run 'dot -Tpng -O d.txt' on Host PC to generate a PNG file."
+	exit 0
+fi
+
 if ! mount|grep -sq '/sys/kernel/debug'; then
 	mount -t debugfs none /sys/kernel/debug
 fi
-- 
1.8.0

